docs/words/index.rst

```diff
@@ -3,8 +3,8 @@
 :LastChangedRevision: $LastChangedRevision$
 :LastChangedBy: $LastChangedBy$
 
-Twisted Words
-=============
+Twisted Words (IRC, XMPP)
```

Comment:
Consistency nit, write this as "IRC and XMPP" like the other lines.

Reply:
Fixed